docs/tasks/tools/install-kubectl.md

```diff
@@ -7,24 +7,22 @@ redirect_from:
 - "/docs/tasks/kubectl/install/"
 - "/docs/tasks/kubectl/install.html"
 ---
+{% capture overview %}
+Use the Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl), to deploy and manage applications on Kubernetes. Using kubectl, you can:
+  * Inspect your cluster resources.
```

Comment:
Put a newline before and after the bullet points so that the markdown is rendered correctly.